.circleci/config.yml

```diff
@@ -4,56 +4,44 @@ version: 2
 jobs:
   python:
     docker:
-      - image: udata/circleci:2-alpine
+      - image: cimg/python:3.7.13
       - image: mongo:3.6
       - image: redis:alpine
     environment:
        BASH_ENV: /root/.bashrc
     steps:
       - checkout
-      - run:
-          name: Compute Python dependencies key
-          command: cat requirements/*.pip > python.deps
       - run:
           name: Get the base reference branch
           command: export BASE_BRANCH=$(base_branch)
       - restore_cache:
           keys:
-          - py3-cache-v9-{{ arch }}-{{ checksum "python.deps" }}
-          - py3-cache-v9-{{ arch }}-{{ .Branch }}
-          - py3-cache-v9-{{ arch }}-{{ .Environment.BASE_BRANCH }}
+          - py3-cache-v2-{{ arch }}-{{ checksum "poetry.lock" }}
+          - py3-cache-v2-{{ arch }}-{{ .Branch }}
+          - py3-cache-v2-{{ arch }}-{{ .Environment.BASE_BRANCH }}
       - run:
           name: Install python dependencies
           command: |
-            virtualenv venv
-            source venv/bin/activate
-            pip install -r requirements/develop.pip
-            pip install -e .
+            poetry install
       - save_cache:
-          key: py3-cache-v9-{{ arch }}-{{ checksum "python.deps" }}
+          key: py3-cache-v2-{{ arch }}-{{ checksum "poetry.lock" }}
           paths:
-          - venv
+            - /home/circleci/.cache/pypoetry/virtualenvs
```

Comment:
We can specify in-project env instead. See https://github.com/etalab/udata-hydra/pull/23/commits/a45524b7403bb850b6b8435d3030f07d2f87f7f5.